### DISCUSSION TITLE
ResNet backbone configs

### DISCUSSION THREAD

**Discussion by TekayaNidham** [open-mmlab/mmrotate#350] [Q&A] [answered]
Hi,
Thank you so much for your work, you saved my Msc thesis!
I have a question about changing resnet backbones, if I want to switch to ResNet 101 or 152, if there other configs i have to changes besides the depth and init_cfg ?

**Comment by liuyanyi** [comment, marked as answer]
I think you should also change neck's input channels

**Reply by TekayaNidham** [reply]
according to this [answer ](https://github.com/open-mmlab/mmdetection/issues/5306#issuecomment-856021461)in mmdetection repo, the input channels stay as they are

**Reply by liuyanyi** [reply]
You are right, that's my mistake. For res101 and res152 the channels are the same. For new backbone it can follow https://mmdetection.readthedocs.io/en/latest/tutorials/how_to.html#get-the-channels-of-a-new-backbone to get channels.
